js/index.js

```diff
@@ -37,6 +37,64 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
+
+//NAV
+
+let nav1 = document.getElementsByTagName('a')[0];
+nav1.textContent = 'Services';
+
+let nav2 = document.getElementsByTagName('a')[1];
+nav2.innerHTML = 'Product';
+
+let nav3 = document.getElementsByTagName('a')[2];
+nav3.innerHTML = 'Vision';
+
+let nav4 = document.getElementsByTagName('a')[3];
+nav4.innerHTML = 'Features';
+
+let nav5 = document.getElementsByTagName('a')[4];
+nav5.innerHTML = 'Contact';
+
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// CTA
+
+let cta = document.getElementsByTagName('h1')[0];
+cta.innerHTML = 'DOM is Awesome';
+
+let button = document.getElementsByTagName('button')[0];
+button.innerHTML = 'Get Started';
+
+// Main Content
+
+let featuresHeader = document.getElementsByTagName('h4')[0];
+featuresHeader.innerHTML = 'Features';
+
+let featuresContent = document.getElementsByTagName('p')[0];
```

Comment:
Try to use the object with all the content for the innerHTML.
from @AislynnEdmiston 's repo:
```
const content = document.getElementsByTagName('p');
content[0].innerHTML = siteContent['main-content']['features-content'];
content[1].innerHTML = siteContent['main-content']['about-content'];
```
She's using bracket notion to access the text so that there's no need to copy & paste 😃